users/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from users.views import *
+urlpatterns = [
+    path('/signup',SignupView.as_view()),
+    path('/signin',SigninView.as_view()),
+]
```

Comment:
-  import 다음에 공백 추가
- import 할 때 *가 아닌 필요한 것들 직접 입력
```suggestion
from users.views import SignupView, SigninView

urlpatterns = [
    path('/signup',SignupView.as_view()),
    path('/signin',SigninView.as_view()),
]
```